README.md

```diff
@@ -4,6 +4,10 @@ Simple tool to validate and lint `metadata.json` files in Puppet modules as
 recommended in Puppet Forge style guidelines from [Puppet forge metadata style
 quide](https://docs.puppetlabs.com/puppet/latest/reference/modules_publishing.html#write-a-metadatajson-file)
 
+## Compatibility
+
+metadata-json-lint is compatible with Ruby versions 2.1.x and 2.3.x.
```

Comment:
2.1.9 is the current ruby version in the AIO and 2.3.1 has been announced as the next version. Is there a platform that ships with 2.2.x? If so, let's definitely add it.

I also need to add 2.0.0 as that's the default for EL7